Source/OxyPlot.SkiaSharp.Wpf/OxySKElement.cs

```diff
@@ -158,5 +158,16 @@ static bool IsPositive(double value)
                 return !double.IsNaN(value) && !double.IsInfinity(value) && value > 0;
             }
         }
+
+        private Window GetAncestorWindowFromVisualTree(DependencyObject startElement)
```

Comment:
Missing comments (per the `PlotView` version)

Reply:
I've omitted the documentation in OxySkElement, because there's nothing documented in the whole file. But I can add it.

Reply:
I hadn't realised; I suppose that's because it's a C&P of the provision by SkiaSharp. It would be nice to include it, given it is a divergence from the 'original' (indeed, we should consider documenting the whole file at some point).